pytorch/training/docker/2.4/py3/Dockerfile.cpu

```diff
@@ -222,6 +222,9 @@ RUN pip install --no-cache-dir -U \
     fastai \
     accelerate \
     # pin numpy requirement for fastai dependency
```

Comment:
nit: let's move this comment down so that it's still with the numpy line? I see this is also the case in SM stages

Reply:
technically spacy thinc and blis dont need to the declared but because we pin numpy, if we dont declare them, numpy<2 will fail. I'll improve the comment to make it clearer.